app/views/people/all_tasks.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= render "shared/header" %>
+
+<div class="persons_tasks">
+
+  <p class="bold"> Finished Tasks </p>
```

Comment:
The way that you've written these two sections require you to loop through the full list of tasks twice. Can you think of a way to do it so you only need to loop through the list one time?